infra/managed-cluster/aks-cluster.bicep

```diff
@@ -23,35 +19,34 @@ param createdDate string = utcNow('yyyy-MM-dd')
 param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
 @description('Required. The parameter object for configuring flux with the aks cluster. The object must contain the fluxCore  and fluxServices values.')
 param fluxConfig object
+@description('Optional. Resource ID of the monitoring log analytics workspace.')
+param monitoringWorkspaceId object
```

Comment:
Can you please change the name to monitoringWorkspace?

Reply:
and descriptions also please

Reply:
Done